qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteDiagnostics.java

```diff
@@ -82,6 +82,8 @@
 
 	private final QuteDiagnosticsForSyntax diagnosticsForSyntax;
 
+	private QuteValidationSettings validationSettings;

```

Comment:
QuteDiagnostic is a singleton. Please remove this field declaration and update java method declartion by adding QuteValidationSettings.